src/pam_oauth2_device.cpp

```diff
@@ -383,6 +383,7 @@ bool is_authorized(Config *config,
         }
         delete[] filter;
     }
+    syslog(LOG_ERR, "cannot find mapping between user %s and local account %s", username_remote, username_local);
```

Comment:
Perhaps LOG_WARNING would be sufficient? I think that LOG_ERR should be used for "unexpected" errors, e.g. network issues, broken config file.

```suggestion
    syslog(LOG_WARNING, "cannot find mapping between user %s and local account %s", username_remote, username_local);
```